[author] [new] Add rollbar.js@1.9.2 w/ git auto-update

jondeandres opened the pull request:
This is the PR to add https://github.com/rollbar/rollbar.js into cdnjs. We've added also all the versions for the existing tags in the repository.

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rollbar/rollbar.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/rollbar-browser
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 112
   - Count of watchers: 25
   - Count of forks: 53
 * NPM download stats (optional):
   - Downloads in the last day: 1,117 
   - Downloads in the last week: 5,751
   - Downloads in the last month: 20,726 

# Essential checklist
 * [x] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

jondeandres commented:
@pvnr0082t added, thanks!

PeterDaveHello commented:
Ping @kennynaoh @x09326

PeterDaveHello commented on ajax/libs/rollbar.js/package.json:
```diff
@@ -0,0 +1,35 @@
+  "name": "rollbar.js",
+  "author": "Rollbar, Inc",
+  "description": "Error tracking and logging from JS to Rollbar",
+  "version": "1.9.1",
```
Looks like it should be 1.9.2?

PeterDaveHello commented:
@jondeandres I wonder if you can just add the latest version so that the review would be easier? Thanks.

jondeandres commented:
@PeterDaveHello I've changed the PR so only 1.9.2 version appears on it. I've also changed the `version` value in `package.json` to `1.9.2`. Thanks for finding that error.

PeterDaveHello commented:
@jondeandres you're welcome! 
@pvnr0082t please help rebase this PR and take a review!

x09326 commented on ajax/libs/rollbar.js/package.json:
```diff
@@ -0,0 +1,35 @@
+    "console-polyfill": "0.2.2",
+    "error-stack-parser": "1.3.3",
+    "extend": "3.0.0"
+  },
```
@jondeandres I think we don't need `dependencies` field in **package.json**, please remove it.

jondeandres commented:
@x09326 @PeterDaveHello I've removed the `dependencies` section in `package.json`. Thanks.
